cmds/core/find/find_go1.16.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.16
+
+package main
+
+import "io/fs"
+import "os"
```

Comment:
Combine these two imports into a single block.

Reply:
that file is gone